### ISSUE TITLE
File Manager changing permissions when saving after edit but not setting them back to what they should be

### ISSUE THREAD

**Issue by meramsey** [litespeedtech/cyberpanel#514]
I noticed that the editor when it opens a file it changes the permissions so it can edit it but does NOT set the previous permissions on the file when saved and closed.

To reproduce set a php file for example to chmod 400
touch testconfig.php
chmod 400 testconfig.php

Then stat the file:
stat testconfig.php

It will look like this

```
[whatthe@wcloud:~/public_html]$ touch testconfig.php
[whatthe@wcloud:~/public_html]$ chmod 400 testconfig.php
[whatthe@wcloud:~/public_html]$ stat testconfig.php
  File: ‘testconfig.php’
  Size: 0               Blocks: 0          IO Block: 4096   regular empty file
Device: fd01h/64769d    Inode: 524577      Links: 1
Access: (0400/-r--------)  Uid: ( 5002/ whatthe)   Gid: ( 5002/ whatthe)
Access: 2021-01-17 00:20:12.321104605 +0000
Modify: 2021-01-17 00:20:12.321104605 +0000
Change: 2021-01-17 00:20:13.543111295 +0000
 Birth: -
[whatthe@wcloud:~/public_html]$
```
When it opens it it does not change it from 400 till saving. It then switches it to 644
```
[whatthe@wcloud:~/public_html]$ stat testconfig.php
  File: ‘testconfig.php’
  Size: 11              Blocks: 8          IO Block: 4096   regular file
Device: fd01h/64769d    Inode: 132395      Links: 1
Access: (0644/-rw-r--r--)  Uid: ( 5002/ whatthe)   Gid: ( 5002/ whatthe)
Access: 2021-01-17 00:22:03.883715607 +0000
Modify: 2021-01-17 00:22:03.883715607 +0000
Change: 2021-01-17 00:22:03.963716045 +0000
 Birth: -
[whatthe@wcloud:~/public_html]$
```

This can be a security concern cause some config files should be more restricted then others and if you don't happen to notice it was changed from saving and not saved and then chmodded back to its original perms you could have a problem. The UI will not show you it changed the permissions unless you manually refresh the filemanager files list.

reproducible on any version C7/Ubuntu latest cyberpanel.

Ideally the permissions value should be noted when accessed/opened and reapplied back after saved.

**Comment by usmannasir** [comment]
Let me check.

**Comment by usmannasir** [comment]
Resolved in v2.0.4-dev, kindly check.
